_includes/payment-orders-resource.md

```diff
@@ -0,0 +1,277 @@
+## Step 1: Create Payment Order
+
+We start by performing a `POST` request towards the `paymentorder` resource
+with the payer information, such as `consumerProfileRef`.
+Remember to read up on our [URL resource][urls].
+
+{% include alert-risk-indicator.md %}
+
+{% include payment-url.md payment_order=true
+when="selecting the payment instrument Vipps or in the 3-D Secure verification
+for Credit Card Payments" %}
+
+### Request
+
+{% include payment-order-purchase.md %}
+
+### Response
+
+The response back should look something like this (abbreviated for brevity):
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/json
+
+{
+    "paymentorder": {
+      "id": "/psp/paymentorders/{{ page.payment_order_id }}"
+    },
+    "operations": [
+        {
+            "href": "{{ page.front_end_url }}/paymentmenu/core/scripts/client/px.paymentmenu.client.js?token={{ page.payment_token }}&culture=sv-SE",
+            "rel": "view-paymentorder",
+            "method": "GET",
+            "contentType": "application/javascript"
+        }
+    ]
+}
+```
+
+{:.table .table-striped}
+| Field          | Type     | Description                                                                        |
+| :------------- | :------- | :--------------------------------------------------------------------------------- |
+| `paymentorder` | `object` | The payment order object.                                                          |
+| └➔&nbsp;`id`   | `string` | {% include field-description-id.md resource="paymentorder" %}                      |
+| `operations`   | `array`  | The array of possible operations to perform, given the state of the payment order. |
+
+The `paymentorder` object is abbreviated since it's just the `id` and
+`operations` we are interested in. Store the `id` of the Payment Order
+in your system to look up status on the completed payment later.
+
+Then find the `view-paymentorder` operation and embed its `href` in a
+`<script>` element.
+That script will then load the Seamless View for the Payment Menu. We will look
+into how to hook that up next.
+
+{% include alert.html type="informative" icon="info" body=" `orderReference` must
+be sent as a part of the `POST` request to `paymentorders` and must represent
+the order ID of the webshop or merchant website." %}
+
+## Step 2: Display the Payment Menu
+
+To load the payment menu from the JavaScript URL obtained in the back end API
+response, it needs to be set as a `script` element's `src` attribute. You can
+cause a page reload and do this with static HTML or you can avoid the page
+refresh by invoking the POST to create the payment order through Ajax and then
+create the script element with JavaScript, all inside the event handler for
+[`onConsumerIdentified`][technical-reference-onconsumer-identified].
```

Comment:
We should have no mention of `onConsumerIdentified` or other Checkin-related events and data in the Payment Menu integration.